src/eureka/S1_detector_processing/ramp_fitting.py

```diff
@@ -87,6 +86,33 @@ def process(self, input):
             Updated for JWST version 1.3.3, code restructure
         '''
         with datamodels.RampModel(input) as input_model:
+
+            if self.s1_meta.grouplevel_bg:
```

Comment:
The simplest way to resolve most of my comments is likely to just replace this line with the following and then remove the ECF settings from the other instruments. That way we're not leading people astray in the templates but get this feature merged as soon as possible:
```
if hasattr(self.s1_meta, 'grouplevel_bg') and self.s1_meta.grouplevel_bg:
```
The above change makes is so that you don't need to add the grouplevel_bg and related settings to the ECF.